README.md

```diff
@@ -3,27 +3,19 @@
 ## Table of contents
 
 - 1.0) [About](#10-about)
-    - 1.1) [Compiling OpenTTD](#11-compiling-openttd)
-    - 1.2) [Contributing to OpenTTD](#12-contributing-to-openttd)
-- 2.0) [Contacting](#20-contacting)
-    - 2.1) [Reporting bugs](#21-reporting-bugs)
-    - 2.2) [Reporting desyncs](#22-reporting-desyncs)
-- 3.0) [Supported platforms](#30-supported-platforms)
-- 4.0) [Installing and running OpenTTD](#40-installing-and-running-openttd)
-    - 4.1) [(Required) 3rd party files](#41-required-3rd-party-files)
-    - 4.2) [OpenTTD directories](#42-openttd-directories)
-    - 4.3) [Portable installations (portable media)](#43-portable-installations-portable-media)
-    - 4.4) [Files in tar (archives)](#44-files-in-tar-archives)
-- 5.0) [OpenTTD features](#50-openttd-features)
-    - 5.1) [Logging of potentially dangerous actions](#51-logging-of-potentially-dangerous-actions)
-    - 5.2) [Frame rate and performance metrics](#52-frame-rate-and-performance-metrics)
-- 6.0) [Configuration file](#60-configuration-file)
-- 7.0) [Translating](#70-translating)
-    - 7.1) [Translation](#71-translation)
-    - 7.2) [Previewing](#72-previewing)
-- 8.0) [Troubleshooting](#80-troubleshooting)
-- 9.0) [Licensing](#90-licensing)
-- X.X) [Credits](#xx-credits)
+    - 1.1) [Downloading OpenTTD](#11-downloading-openttd)
+    - 1.2) [OpenTTD gameplay manual](#12-openttd-gameplay-manual)
+    - 1.3) [Supported platforms](#13-supported-platforms)
+    - 1.4) [Installing and running OpenTTD](#14-installing-and-running-openttd)
+    - 1.5) [Add-on content / mods](#15-add-on-content--mods)
+    - 1.6) [OpenTTD directories](#16-openttd-directories)
+    - 1.7) [Compiling OpenTTD](#17-compiling-openttd)
+- 2.0) [Contact and community](#20-contact-and-community)
+    - 2.1) [Contributing to OpenTTD](#21-contributing-to-openttd)
+    - 2.2) [Reporting bugs](#22-reporting-bugs)
+    - 2.3) [Translating](#23-translating)
+- 3.0) [Licensing](#30-licensing)
+- 4.0) [Credits](#40-credits)
 
 ## 1.0) About
 
```

Comment:
"originally written by" (i can't comment on the line itself, grr, github). Section wants its line breaks... broken into the new style as well.